### PR TITLE
Feat: added utilities and text utilities

### PR DESCRIPTION
**Pull request by vivekmehto**
image

### CODE REVIEW

**Review by aishwarya-bm — APPROVED** [review]
So many variants, looks good!

File: components/utilities/utilities.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  border-radius: 5px;
+}
+
+/* medium  */
+.br-m {
+  border-radius: 10px;
+}
+
+/* large  */
+.br-l {
```
Comment (krishanu2911) [inline components/utilities/utilities.css]: Just a suggestion class name br-l is for border-radius but if changed to br-rad-lg will be easy to understand

**Review by krishanu2911 — COMMENTED** [review]
everything is good to go just added a small suggestion

File: components/utilities/utilities.css (hunk 1 above)
Comment (vivekmehto) [reply components/utilities/utilities.css]: changed it. Thank You

**Review by krishanu2911 — APPROVED** [review]

**Pull request merged**